safe-global/web-core
# style: use Figma icons

## What it solves

Resolves #733

## How this PR fixes it

All icons used on Figma (that are not from MUI) now replace those which were from MUI

## Execeptions

### Icon issues

- Safe apps icons: the bordered-bookmark icon fills, despite only specifying a `stroke`. The standard icon, in `border` colour, is currently used in it's place. The share icon has a filled circle on the left as well?

![image](https://user-images.githubusercontent.com/20442784/193026999-0e7b6607-9e57-4c3e-aa4c-c74d2b1b4c71.png)
![image](https://user-images.githubusercontent.com/20442784/193027064-ce8c31dc-e766-4661-be97-9cadc59a379d.png)

- The execution rocket `fill`s, whereas it should only `stroke`

![image](https://user-images.githubusercontent.com/20442784/193026143-d86664b9-bcbd-4789-9804-ba4c132abea1.png)
![image](https://user-images.githubusercontent.com/20442784/193026195-26d176b3-27c2-458b-8677-193296e7b5e3.png)

## How to test it

Navigate the Safe and observe the new icons, matching that of the Figma designs (and no visual inconsistencies).
